table/queries/merged_reduced_scans.sql

```diff
@@ -38,7 +38,7 @@ CREATE OR REPLACE TABLE `PROJECT_NAME.DERIVED_DATASET.merged_reduced_scans_v2`
 PARTITION BY date
 # Columns `source` and `country_name` are always used for filtering and must come first.
 # `network` and `domain` are useful for filtering and grouping.
```

Comment:
It may be worth mentioning that the order matters and link to https://cloud.google.com/bigquery/docs/clustered-tables#cluster_column_ordering
Also that we need domain to come after country to benefit from the default domain selection.